### PR TITLE
"Access Child Components From the Template" section - angular style guide update

### PR DESCRIPTION
**Pull request by alejochang**
Adding examples for existing plunkers.

### CODE REVIEW

File: examples/components/components-ngFor/app/index.ts [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-    BrowserModule
-  ],
-  declarations: [
-    App
-  ],
-  bootstrap: [ App ]
-})
-export class AppModule {
-}
-platformBrowserDynamic().bootstrapModule(AppModule);
```
Comment (ggprod) [inline examples/components/components-ngFor/app/index.ts]: why is this file in the NgFor section being removed?

File: examples/components/components-access-child/index.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   </head>
   <body>
-    <app>Loading...</app>
+    <rio-app>Loading...</rio-app>
 
     <script>
-      System.import('app');
+      System.import('main');
     </script>
   </body>
```
Comment (ggprod) [inline examples/components/components-access-child/index.html]: why is this file in the NgFor section being modified?

File: examples/components/components-access-child-ngform/system.config.js [hunk 3]
```diff
@@ -27,18 +27,20 @@ System.config({
...
+    "main": "main.ts"
   },
 
   packages: {
     'app': {
-      defaultExtension: 'ts',
-      main: './index.ts'
+      defaultExtension: 'ts'
     }
   }
```
Comment (ggprod) [inline examples/components/components-access-child-ngform/system.config.js]: why is this file in the NgFor section being modified?

File: examples/components/components-ngFor/app/index.ts (hunk 1 above)
Comment (alejochang) [inline examples/components/components-ngFor/app/index.ts]: When refactoring, this file was renamed/relocated as main.ts in the root folder.

File: examples/components/components-ngFor/app/app.component.ts [hunk 4]
```diff
@@ -1,38 +0,0 @@
...
-})
-export class App {
-  items: string[];
-  
-  constructor() {
-    this.items = ['Spectacles', 'Giraffe', 'Turtle', 'Shark', 'Lamp', 'Chocolate', 'Beef', 'Drawer', 'Brocolli', 'Tomato', 'Plate', 'Zebra'];
-  }
-  
-}
-
```
Comment (alejochang) [inline examples/components/components-ngFor/app/app.component.ts]: This `ngFor` folder wasn't used. The current `NgFor` section is under the `directives` folder.

File: examples/components/components-access-child/index.html (hunk 2 above)
Comment (alejochang) [inline examples/components/components-access-child/index.html]: For consistency, the main app file was renamed as `main.ts`
In the `system.config.js` file, that file was mapped as `main`, therefore it was changed the import call in index.html

File: examples/components/components-access-child-ngform/system.config.js (hunk 3 above)
Comment (alejochang) [inline examples/components/components-access-child-ngform/system.config.js]: This file isn't part of the NgFor section, is for the example that uses NgForm for the "Access Child Components From the Template" section.

File: examples/components/components-ngFor/app/app.component.ts (hunk 4 above)
Comment (ggprod) [inline examples/components/components-ngFor/app/app.component.ts]: ah, I see, ok

File: examples/components/components-ngFor/app/index.ts (hunk 1 above)
Comment (ggprod) [inline examples/components/components-ngFor/app/index.ts]: ok

File: examples/components/components-access-child/index.html (hunk 2 above)
Comment (ggprod) [inline examples/components/components-access-child/index.html]: ok

File: examples/components/components-access-child-ngform/system.config.js (hunk 3 above)
Comment (ggprod) [inline examples/components/components-access-child-ngform/system.config.js]: ok

**Comment by bennett000** [comment]
:shipit:

**Pull request merged**